src/main/scala/piconot/app.scala

```diff
@@ -0,0 +1,69 @@
+/*
+Written by Ben Wiedermann
+https://github.com/hmc-cs111-spring2023/hw-7-sample-solution/blob/main/src/main/scala/piconot/external/piconot.scala
+*/
+
```

Comment:
I know this file is from prof Wiedermann but it has nice error handling! good use of his resource and giving credit to it.